app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -211,6 +214,49 @@ class GithubWebhookSubscription extends SubscriptionHandler {
     }
   }
 
+  /// Handles a GitHub webhook with the event type "merge_group".
+  ///
+  /// Unlike a pull request a merge group contains multiple commits created
+  /// from multiple squashed pull requests. A merge group is therefore not
```

Comment:
is this part true? the "branch" is actually a squash of all PRs at that point?  My understanding was there was a merge_group created and checks requested for each PR in the merge queue; with CI running for each request in the merge queue.

Reply:
I'll reword it. Each PR is squashed individually and produces one commit, then that commit is stacked on top of other commits in the merge group. So a group will contain multiple commits, one per PR. There's no extra squashing done.

Reply:
But we fire of CI for each PR?